buildspec/buildspec_secscan.yaml

```diff
@@ -10,7 +10,6 @@ phases:
        - apt-get install -y wget apt-transport-https gnupg lsb-release curl
        - curl -OL https://github.com/aws/aws-codeguru-cli/releases/download/0.2.3/aws-codeguru-cli.zip
        - unzip aws-codeguru-cli.zip
-       - export PATH=$PATH:./aws-codeguru-cli/bin
```

Comment:
Not needed as command is executed with full path `$BASE/aws-codeguru-cli/bin/aws-codeguru-cli`